blacklight-cornell/app/views/browse/_pagination.html.erb

```diff
@@ -17,35 +17,34 @@
       <% if !@has_previous %>
         &laquo; Previous |
       <% elsif params[:order] != "reverse" %>
-        <%= link_to "/browse?authq=" + params[:authq] + "&start=0" +"&browse_type=" + params[:browse_type] + '&order=reverse' + location do %>
+        <%= link_to browse_index_path(authq: params[:authq], browse_type: params[:browse_type], start: 0, order: "reverse") + location do %>
```

Comment:
If you're up to cleaning up just a bit more, you could just add in the fq parameter to the browse_index_paths and get rid of that `location` variable.